packages/@uppy/companion/src/companion.js

```diff
@@ -237,7 +237,7 @@ const validateConfig = (companionOptions) => {
     fs.accessSync(`${companionOptions.filePath}`, fs.R_OK | fs.W_OK) // eslint-disable-line no-bitwise
   } catch (err) {
     throw new Error(
-      `No access to "${companionOptions.filePath}". Please ensure the directory exists and with read/write permissions.`
+      `No access to "${companionOptions.filePath}". Please ensure the directory exists and with read/write permissions.`,
```

Comment:
Not sure about more commas, is this for the same reason, to add new args easier?

Reply:
Yes, less diff noise if args are added or removed from functions. Same as for object literals, but in v1 function argument comma enforcement was disabled